restli-server/src/main/java/com/linkedin/restli/internal/server/response/RestLiResponseHandler.java

```diff
@@ -63,57 +64,12 @@
   private final MethodAdapterRegistry _methodAdapterRegistry;
   private final ErrorResponseBuilder _errorResponseBuilder;
 
-  public RestLiResponseHandler()
```

Comment:
This is backward-incompatible. Please deprecate the existing constructors, and have them defer to the new classes while also maintaining existing behavior. Please leave a javadoc explaining the suggested alternative for the user.